aws/cluster/templates/cluster-spec.yaml

```diff
@@ -23,22 +23,23 @@ ${cloud-labels}
 ${etcd-clusters}
   keyStore: s3://${kops-state-bucket}/${cluster-name}/pki
   kubeAPIServer:
-    address: 127.0.0.1
-    admissionControl:
+    insecureBindAddress: 127.0.0.1
+    enableAdmissionPlugins:
     - NamespaceLifecycle
     - LimitRanger
     - ServiceAccount
     - PersistentVolumeLabel
     - DefaultStorageClass
     - DefaultTolerationSeconds
-    - Initializers
-    - NodeRestriction
-    - Priority
     - DenyEscalatingExec
-    - PodPreset
```

Comment:
The order was important actually

Reply:
From https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/

> For Kubernetes version 1.10 and later, we recommend running the following set of admission controllers using the --enable-admission-plugins flag (order doesn’t matter).